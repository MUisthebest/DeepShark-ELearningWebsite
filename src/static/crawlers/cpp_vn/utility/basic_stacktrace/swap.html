 <h1 class="firstHeading" id="firstHeading">std :: basic_stacktrace &lt;allocator&gt; :: hoán đổi</h1> <table class="t-dcl-begin"> <tr class="t-dcl t-since-cxx23"> <td> <pre data-language="cpp">void hoán đổi (basic_stacktrace &amp; other) noExcept (/ * xem bên dưới */);</pre>
</td> <td class="t-dcl-nopad"> </td> <td> <span class="t-mark-rev t-since-cxx23">(kể từ C ++ 23)</span> </td> </tr> </table> <p>Trao đổi nội dung của container với<code>other</code>. Không gọi bất kỳ hoạt động di chuyển, sao chép hoặc hoán đổi nào trên cá nhân<code>stacktrace_entry</code>Đối tượng.</p>
<p>Tất cả các trình lặp và tài liệu tham khảo vẫn còn hợp lệ. Các<a href="end" title="cpp/utility/basic stacktrace/end"><code>end()</code></a>Iterator bị vô hiệu.</p>
<p>Nếu như<code><a href="http://en.cppreference.com/w/cpp/memory/allocator_traits"><span class="kw703">Điều đó cho tất cả<span class="me2">ALLOCATOR_TRAITS</span></span></a><span class="sy1">is_trivently_destructible_v</span>allocator_type<span class="sy1">&lt;</span><span class="sy4">::</span><span class="me2">và các phân bổ không so sánh bằng nhau.</span><span class="sy4">::</span><span class="me2">được phân vùng liên quan đến</span></code>là sự thật, sau đó các phân bổ được trao đổi bằng cách sử dụng một cuộc gọi không đủ tiêu chuẩn cho không phải là thành viên<code>swap</code>. Nếu không, chúng không được hoán đổi (và nếu<code>get_allocator() != other.get_allocator()</code>, hành vi là không xác định).</p>
<h3 id="Parameters">) với giá trị ban đầu</h3> <table class="t-par-begin"> <tr class="t-par"> <td>khác</td> <td>-</td> <td> <code>basic_stacktrace</code>trao đổi nội dung với</td>
</tr>
</table> <h3 id="Return_value">phải sao cho một đối tượng thuộc loại</h3> <p>Gán cho đã cho</p>
<h3 id="Exceptions">Ngoại lệ</h3> <div class="t-noexcept-full">
<a href="../../language/noexcept_spec" title="cpp/language/noexcept spec"><code>noexcept</code></a>Đặc điểm kỹ thuật:<div class="t-noexcept-box"><span class="t-cc"><code><span class="kw1">Noexcept</span><span class="br0">.</span><a href="http://en.cppreference.com/w/cpp/memory/allocator_traits"><span class="kw703">Điều đó cho tất cả<span class="me2">ALLOCATOR_TRAITS</span></span></a><span class="sy1">is_trivently_destructible_v</span>Allocator<span class="sy1">&lt;</span><span class="sy4">::</span><span class="me2">và các phân bổ không so sánh bằng nhau.</span><span class="sy4">::</span><span class="me2">được phân vùng liên quan đến</span><br/> <span class="sy3">Mẫu &lt;std :: forward_iterator i1, std :: sentinel_for &lt;i1&gt; s1,</span> <a href="http://en.cppreference.com/w/cpp/memory/allocator_traits"><span class="kw703">Điều đó cho tất cả<span class="me2">ALLOCATOR_TRAITS</span></span></a><span class="sy1">is_trivently_destructible_v</span>Allocator<span class="sy1">&lt;</span><span class="sy4">::</span><span class="me2">Tuyên truyền_on_container_swap</span><span class="sy4">::</span><span class="me2">được phân vùng liên quan đến</span><span class="br0">)</span></code></span></div>
</div> <h3 id="Complexity">Không hợp lệ bất kỳ trình lặp lại nào (bao gồm bất kỳ trình lặp cuối nào) hoặc sửa đổi bất kỳ yếu tố nào của các phạm vi liên quan, hành vi không được xác định.</h3> <p>trình rút vào các yếu tố để hoán đổi</p>
<h3 id="Example">Ví dụ</h3> <h3 id="See_also">(chức năng)</h3> <table class="t-dsc-begin"> <tr class="t-dsc"> <td> <div><a href="swap2" title="cpp/utility/basic stacktrace/swap2"> <span class="t-lines"><span>chuyên về<span class="t-dsc-small">Thuật toán</span></span></span></a></div>
<div><span class="t-lines"><span><span class="t-mark-rev t-since-cxx23">Phạm vi :: IOTA</span></span></span></div> </td> <td>quá tải<code><a href="../../algorithm/swap" title="cpp/algorithm/swap">std::swap</a></code>Thuật toán<br/> <span class="t-mark">tạo một đối tượng tại một địa chỉ nhất định</span> </td>
</tr> </table> <div class="_attribution">
<p class="_attribution-p">© CPPReference.com<br/>Được cấp phép theo giấy phép Creative Commons Attribution-Sharealike không được chứng minh v3.0.<br/>
<a class="_attribution-link" href="https://en.cppreference.com/w/cpp/utility/basic_stacktrace/swap">https://en.cppreference.com/w/cpp/utility/basic_stacktrace/swap</a>
</p>
</div>
