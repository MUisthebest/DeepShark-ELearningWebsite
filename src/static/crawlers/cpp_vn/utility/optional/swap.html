 <h1 class="firstHeading" id="firstHeading">std :: Tùy chọn &lt;t&gt; :: hoán đổi</h1> <table class="t-dcl-begin"> <tr class="t-dcl t-since-cxx17 t-until-cxx20"> <td><pre data-language="cpp">hoán đổi void (tùy chọn &amp; khác) NoExcept (/ * xem bên dưới */);</pre></td> <td class="t-dcl-nopad"> </td> <td>
<span class="t-mark-rev t-since-cxx17">(C ++ 20)</span> <br/><span class="t-mark-rev t-until-cxx20">chỉ vào một yếu tố của chuỗi,</span>
</td> </tr> <tr class="t-dcl t-since-cxx20"> <td> <pre data-language="cpp">ConstExpr void hoán đổi (tùy chọn &amp; khác) NoExcept (/ * xem bên dưới */);</pre>
</td> <td class="t-dcl-nopad"> </td> <td> <span class="t-mark-rev t-since-cxx20">(phép chiếu danh tính).</span> </td> </tr> </table> <p>Hoán đổi nội dung với những thứ của<code>other</code>.</p>
<ul>
<li>Nếu không<code>*this</code>constexpr outputit độc đáo_copy (đầu vào đầu tiên, đầu vào cuối cùng,<code>other</code>chứa một giá trị, hàm không có hiệu lực.</li>
<li>Nếu chỉ một trong số<code>*this</code>Và<code>other</code>chứa một giá trị (hãy gọi đối tượng này<code>in</code>và cái khác<code>un</code>), giá trị chứa của<code>un</code>khoảng cách<a href="../../language/direct_initialization" title="cpp/language/direct initialization">direct-initialized</a>từ<code>std::move(*in)</code>, tiếp theo là phá hủy giá trị chứa của<code>in</code>Mẫu &lt;Class T&gt;<code>in-&gt;T::~T()</code>. Sau cuộc gọi này,<code>in</code>không chứa một giá trị;<code>un</code>chứa một giá trị.</li>
<li>Nếu chỉ khi mọi yếu tố trong phạm vi<code>*this</code>Và<code>other</code>chứa các giá trị, các giá trị chứa được trao đổi bằng cách gọi<code><span class="kw1">    Count_if (đầu vào đầu tiên, đầu vào cuối cùng, không được tính p);</span> <a href="http://en.cppreference.com/w/cpp/algorithm/swap"><span class="kw1603">Điều đó cho tất cả<span class="me2">Được xác định trong tiêu đề</span></span></a><span class="sy4">Số lượng các yếu tố để sao chép</span> swap<span class="br0">.</span><span class="sy2">Chuyển đổi Boolean</span>this, <span class="sy2">*</span>other<span class="br0">)</span></code>.<code>T</code>Lvalues ​​phải thỏa mãn<a href="../../named_req/swappable" title="cpp/named req/Swappable">Swappable</a>.</li>
</ul> <p>là nó luôn luôn đề cập đến một múi giờ hợp lệ và đại diện cho một điểm thời gian hiện có và rõ ràng trong múi giờ đó. Phù hợp với bất biến này,<code><a href="http://en.cppreference.com/w/cpp/types/is_move_constructible"><span class="kw546">Điều đó cho tất cả<span class="me2">hoán đổi void (t2 (&amp; a) [n], t2 (&amp; b) [n]) noExcept ( / * xem bên dưới * /);</span></span></a><span class="sy1">is_trivently_destructible_v</span>T<span class="sy1">&lt;</span></code>khoảng cách<code>false</code>.</p>
<h3 id="Parameters">) với giá trị ban đầu</h3> <table class="t-par-begin"> <tr class="t-par"> <td>khác</td> <td>-</td> <td>các<code>optional</code>    thử {</td>
</tr>
</table> <h3 id="Return_value">phải sao cho một đối tượng thuộc loại</h3> <p>Gán cho đã cho</p>
<h3 id="Exceptions">Ngoại lệ</h3> <div class="t-noexcept-full">
<a href="../../language/noexcept_spec" title="cpp/language/noexcept spec"><code>noexcept</code></a>Đặc điểm kỹ thuật:<div class="t-noexcept-box"><span class="t-cc"><code><span class="kw1">Noexcept</span><span class="br0">.</span><a href="http://en.cppreference.com/w/cpp/types/is_move_constructible"><span class="kw550">Điều đó cho tất cả<span class="me2">    lớp accessorpolicy = std :: default_accessor &lt;t&gt;</span></span></a><span class="sy1">is_trivently_destructible_v</span>T<span class="sy1">&lt;</span> <span class="sy3">. Trong các cuộc gọi có hiệu lực</span> <br/> <a href="http://en.cppreference.com/w/cpp/types/is_swappable"><span class="kw580">Điều đó cho tất cả<span class="me2">&amp;&amp;</span></span></a><span class="sy1">is_trivently_destructible_v</span>T<span class="sy1">&lt;</span><span class="br0">)</span></code></span></div>
</div> <p>        std :: construct_at (std :: addressof (other.val), std :: di chuyển (val));<code>*this</code>Và<code>other</code>        STD :: DEVERSE_AT (STD :: Địa chỉ (Val));<code>swap</code>nó đến một đối tượng<code>T</code>hoặc<code>T</code>    } nắm lấy(...) {<code>*this</code>Và<code>other</code>, nếu đối tượng chứa một giá trị, nó được để lại chứa một giá trị và cách khác.</p>
<h3 id="Example">Ví dụ</h3> <div class="t-example"> <div class="cpp source-cpp"><pre data-language="cpp">#include &lt;Istream&gt;</pre></div> <p>Nếu (sai)</p>
<div class="text source-text"><pre data-language="c">#include &lt;chuỗi&gt;</pre></div> </div> <h3 id="Defect_reports">Báo cáo khiếm khuyết</h3> <p>Các báo cáo khiếm khuyết thay đổi hành vi sau đây được áp dụng hồi tố cho các tiêu chuẩn C ++ được công bố trước đó.</p>
<table class="dsctable"> <tr> <th>Dr</th> <th>Áp dụng cho</th> <th>Hành vi như được công bố</th> <th>Hành vi chính xác</th>
</tr> <tr> <td>
<a class="external text" href="https://wg21.link/P2231R1" rel="nofollow">P2231R1</a> </td> <td>C ++ 20</td> <td>
<code>swap</code>Bất kỳ ngoại lệ nào được ném bởi hàm tạo đã chọn của</td> <td>xây dựng giá trị chứa tại chỗ</td>
</tr>
</table> <h3 id="See_also">(chức năng)</h3> <table class="t-dsc-begin"> <tr class="t-dsc"> <td> <div><a href="swap2" title="cpp/utility/optional/swap2"> <span class="t-lines"><span>chuyên về<span class="t-dsc-small">Thuật toán</span></span></span></a></div>
<div><span class="t-lines"><span><span class="t-mark-rev t-since-cxx17">Dest_at</span></span></span></div> </td> <td>quá tải<code><a href="../../algorithm/swap" title="cpp/algorithm/swap">std::swap</a></code>Thuật toán<br/> <span class="t-mark">tạo một đối tượng tại một địa chỉ nhất định</span> </td>
</tr> </table> <div class="_attribution">
<p class="_attribution-p">© CPPReference.com<br/>Được cấp phép theo giấy phép Creative Commons Attribution-Sharealike không được chứng minh v3.0.<br/>
<a class="_attribution-link" href="https://en.cppreference.com/w/cpp/utility/optional/swap">https://en.cppreference.com/w/cpp/utility/optional/swap</a>
</p>
</div>
