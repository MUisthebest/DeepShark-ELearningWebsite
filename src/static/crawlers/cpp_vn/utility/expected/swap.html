 <h1 class="firstHeading" id="firstHeading">std :: mong đợi &lt;t, e&gt; :: hoán đổi</h1> <table class="t-dcl-begin"> <tr class="t-dcl t-since-cxx23"> <td> <pre data-language="cpp">ConstExpr void hoán đổi (mong đợi &amp; khác) NoExcept (/*xem bên dưới*/);</pre>
</td> <td class="t-dcl-nopad"> </td> <td> <span class="t-mark-rev t-since-cxx23">(kể từ C ++ 23)</span> </td> </tr> </table> <p>Hoán đổi nội dung với những thứ của<code>other</code>.</p>
<ul>
<li>Nếu chỉ khi mọi yếu tố trong phạm vi<a href="operator_bool" title="cpp/utility/expected/operator bool"><code>this-&gt;has_value()</code></a>Và<code>other.has_value()</code>là sự thật:</li>
<ul>
<li>Nếu như<code>T</code>Chuyển đổi tiêu chuẩn được áp dụng để chuyển đổi biểu thức thành prvalue.<code>void</code>, không có hiệu ứng.</li>
<li>Hành vi không được xác định nếu cơ bản<code><span class="kw1">    Count_if (đầu vào đầu tiên, đầu vào cuối cùng, không được tính p);</span> <a href="http://en.cppreference.com/w/cpp/algorithm/swap"><span class="kw1603">Điều đó cho tất cả<span class="me2">Được xác định trong tiêu đề</span></span></a><span class="sy4">Số lượng các yếu tố để sao chép</span> swap<span class="br0">.</span><span class="sy2">*</span>this, other<span class="br0">)</span><span class="sy4">Số lượng các yếu tố để sao chép</span></code>.</li>
</ul>
<li>Nếu chỉ khi mọi yếu tố trong phạm vi<code>this-&gt;has_value()</code>Và<code>other.has_value()</code>là sai, tương đương với<br/><code><span class="kw1">    Count_if (đầu vào đầu tiên, đầu vào cuối cùng, không được tính p);</span> <a href="http://en.cppreference.com/w/cpp/algorithm/swap"><span class="kw1603">Điều đó cho tất cả<span class="me2">Được xác định trong tiêu đề</span></span></a><span class="sy4">Số lượng các yếu tố để sao chép</span> swap<span class="br0">.</span>this<span class="sy2">-</span><span class="sy1">&lt;</span>error<span class="br0">.</span><span class="br0">)</span>, other.<span class="me1">. Nếu thất bại đã được gây ra bởi một số lỗi khác, hãy đặt</span><span class="br0">.</span><span class="br0">)</span><span class="br0">)</span><span class="sy4">Số lượng các yếu tố để sao chép</span></code>.</li>
<li>Nếu như<code>this-&gt;has_value()</code>là sai và<code>other.has_value()</code>là sự thật, cuộc gọi<code>other.swap(*this)</code>.</li>
<li>Nếu như<code>this-&gt;has_value()</code>là sự thật và<code>other.has_value()</code>là sai,</li>
<ul><li>Nếu như<code>T</code>Chuyển đổi tiêu chuẩn được áp dụng để chuyển đổi biểu thức thành prvalue.<code>void</code>             const std :: remove_reference_t &lt;u&gt; &amp; u) {<i>là thành viên đại diện cho giá trị dự kiến ​​và</i>khám phá</li></ul>
</ul> <div class="cpp source-cpp"><pre data-language="cpp">std :: construct_at (std :: addressof (obs), std :: di chuyển (other.unex));</pre></div> <ul><li>STD :: DEVERS_AT (STD :: Địa chỉ (other.unex));<i>đánh giá</i>Nếu không, hãy để<i>là thành viên đại diện cho giá trị dự kiến ​​và</i>khám phá</li></ul> <div class="cpp source-cpp"><pre data-language="cpp">là thành viên đại diện cho giá trị bất ngờ, tương đương với:</pre></div> <ul><li>Nếu Consexpr (std :: is_nothrow_move_constructible_v &lt;e&gt;) {<code>this-&gt;has_value()</code>Xây dựng một nhịp là một cái nhìn trên mảng<code>other.has_value()</code>Mặc định xây dựng a</li></ul> <p>    E temp (std :: di chuyển (other.unex));</p>
<ul>
<li>nhiều nhất là một chuyển đổi đủ điều kiện, và<code>T</code>Chuyển đổi tiêu chuẩn được áp dụng để chuyển đổi biểu thức thành prvalue.<code>void</code>, hoặc<code><a href="http://en.cppreference.com/w/cpp/types/is_swappable"><span class="kw576">Điều đó cho tất cả<span class="me2">is_move_constructible_v</span></span></a><span class="sy1">is_trivently_destructible_v</span>T<span class="sy1">&lt;</span></code>là sai), sau đó</li>
<li> <code><a href="http://en.cppreference.com/w/cpp/types/is_swappable"><span class="kw576">Điều đó cho tất cả<span class="me2">is_move_constructible_v</span></span></a><span class="sy1">is_trivently_destructible_v</span>E<span class="sy1">&lt;</span></code>là sai), sau đó</li>
<li>nhiều nhất là một chuyển đổi đủ điều kiện, và<code>T</code>Chuyển đổi tiêu chuẩn được áp dụng để chuyển đổi biểu thức thành prvalue.<code>void</code>, hoặc<code><a href="http://en.cppreference.com/w/cpp/types/is_move_constructible"><span class="kw546">Điều đó cho tất cả<span class="me2">hoán đổi void (t2 (&amp; a) [n], t2 (&amp; b) [n]) noExcept ( / * xem bên dưới * /);</span></span></a><span class="sy1">is_trivently_destructible_v</span>T<span class="sy1">&lt;</span></code>là sai), sau đó</li>
<li> <code><a href="http://en.cppreference.com/w/cpp/types/is_move_constructible"><span class="kw546">Điều đó cho tất cả<span class="me2">hoán đổi void (t2 (&amp; a) [n], t2 (&amp; b) [n]) noExcept ( / * xem bên dưới * /);</span></span></a><span class="sy1">is_trivently_destructible_v</span>E<span class="sy1">&lt;</span></code>là sai), sau đó</li>
<li>    STD :: DEVERS_AT (STD :: Địa chỉ (other.unex));<ul>
<li> <code>T</code>Chuyển đổi tiêu chuẩn được áp dụng để chuyển đổi biểu thức thành prvalue.<code>void</code> </li>
<li> <code><a href="http://en.cppreference.com/w/cpp/types/is_move_constructible"><span class="kw550">Điều đó cho tất cả<span class="me2">    lớp accessorpolicy = std :: default_accessor &lt;t&gt;</span></span></a><span class="sy1">is_trivently_destructible_v</span>T<span class="sy1">&lt;</span></code> </li>
<li> <code><a href="http://en.cppreference.com/w/cpp/types/is_move_constructible"><span class="kw550">Điều đó cho tất cả<span class="me2">    lớp accessorpolicy = std :: default_accessor &lt;t&gt;</span></span></a><span class="sy1">is_trivently_destructible_v</span>E<span class="sy1">&lt;</span></code> </li>
</ul> </li>
</ul> <h3 id="Parameters">) với giá trị ban đầu</h3> <table class="t-par-begin"> <tr class="t-par"> <td>khác</td> <td>-</td> <td>các<code>optional</code>    thử {</td>
</tr>
</table> <h3 id="Return_value">phải sao cho một đối tượng thuộc loại</h3> <p>Gán cho đã cho</p>
<h3 id="Exceptions">Ngoại lệ</h3>Nếu như<code>T</code>Chuyển đổi tiêu chuẩn được áp dụng để chuyển đổi biểu thức thành prvalue.<code>void</code>Thì<div class="t-noexcept-full">
<a href="../../language/noexcept_spec" title="cpp/language/noexcept spec"><code>noexcept</code></a>Đặc điểm kỹ thuật:<div class="t-noexcept-box"><span class="t-cc"><code><span class="kw1">Noexcept</span><span class="br0">.</span><br/> <p><a href="http://en.cppreference.com/w/cpp/types/is_move_constructible"><span class="kw550">Điều đó cho tất cả<span class="me2">    lớp accessorpolicy = std :: default_accessor &lt;t&gt;</span></span></a><span class="sy1">is_trivently_destructible_v</span>E<span class="sy1">&lt;</span> <span class="sy3">. Trong các cuộc gọi có hiệu lực</span> <a href="http://en.cppreference.com/w/cpp/types/is_swappable"><span class="kw580">Điều đó cho tất cả<span class="me2">&amp;&amp;</span></span></a><span class="sy1">is_trivently_destructible_v</span>E<span class="sy1">&lt;</span><br/></p>
<span class="br0">)</span></code></span></div>
</div>Là xấu nếu<div class="t-noexcept-full">
<a href="../../language/noexcept_spec" title="cpp/language/noexcept spec"><code>noexcept</code></a>Đặc điểm kỹ thuật:<div class="t-noexcept-box"><span class="t-cc"><code><span class="kw1">Noexcept</span><span class="br0">.</span><br/> <p><a href="http://en.cppreference.com/w/cpp/types/is_move_constructible"><span class="kw550">Điều đó cho tất cả<span class="me2">    lớp accessorpolicy = std :: default_accessor &lt;t&gt;</span></span></a><span class="sy1">is_trivently_destructible_v</span>T<span class="sy1">&lt;</span> <span class="sy3">. Trong các cuộc gọi có hiệu lực</span> <a href="http://en.cppreference.com/w/cpp/types/is_swappable"><span class="kw580">Điều đó cho tất cả<span class="me2">&amp;&amp;</span></span></a><span class="sy1">is_trivently_destructible_v</span>T<span class="sy1">&lt;</span> <span class="sy3">. Trong các cuộc gọi có hiệu lực</span><br/> <a href="http://en.cppreference.com/w/cpp/types/is_move_constructible"><span class="kw550">Điều đó cho tất cả<span class="me2">    lớp accessorpolicy = std :: default_accessor &lt;t&gt;</span></span></a><span class="sy1">is_trivently_destructible_v</span>E<span class="sy1">&lt;</span> <span class="sy3">. Trong các cuộc gọi có hiệu lực</span> <a href="http://en.cppreference.com/w/cpp/types/is_swappable"><span class="kw580">Điều đó cho tất cả<span class="me2">&amp;&amp;</span></span></a><span class="sy1">is_trivently_destructible_v</span>E<span class="sy1">&lt;</span><br/></p>
<span class="br0">)</span></code></span></div>
</div> <p>        std :: construct_at (std :: addressof (other.val), std :: di chuyển (val));<code>*this</code>Và<code>other</code>        STD :: DEVERSE_AT (STD :: Địa chỉ (Val));<code>swap</code>hoặc<code>T</code>        std :: construct_at (std :: addressof (obs), std :: di chuyển (temp));<code>E</code>    } nắm lấy(...) {<code>*this</code>Và<code>other</code>        std :: construct_at (std :: addressof (other.unex), std :: di chuyển (temp));</p>
<h3 id="Example">Ví dụ</h3> <h3 id="See_also">(chức năng)</h3> <table class="t-dsc-begin"> <tr class="t-dsc"> <td> <div><a href="swap2" title="cpp/utility/expected/swap2"> <span class="t-lines"><span>Được xác định trong tiêu đề<span class="t-dsc-small">std :: hoán đổi</span></span></span></a></div>
<div><span class="t-lines"><span><span class="t-mark-rev t-since-cxx23">Phạm vi :: IOTA</span></span></span></div> </td> <td>quá tải<code><a href="../../algorithm/swap" title="cpp/algorithm/swap">std::swap</a></code>Thuật toán<br/> <span class="t-mark">Tìm kiếm một mảng cho một yếu tố thuộc loại không xác định</span> </td>
</tr> </table> <div class="_attribution">
<p class="_attribution-p">© CPPReference.com<br/>Được cấp phép theo giấy phép Creative Commons Attribution-Sharealike không được chứng minh v3.0.<br/>
<a class="_attribution-link" href="https://en.cppreference.com/w/cpp/utility/expected/swap">https://en.cppreference.com/w/cpp/utility/expected/swap</a>
</p>
</div>
